### PR TITLE
Unit Test Cases

### PR DESCRIPTION
**Pull request by AbhijitP98**

### CODE REVIEW

File: src/sections/Header.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    test("Check the logo img of alt logo", () => {
+        render(
+            <Header />
+        )
+        expect(screen.getByAltText('logo')).toBeInTheDocument();
+    })
+
+    test("Check text with Don't have an account should be rendered", () => {
+        render(
+            <Header />
```
Comment (NaveenReliable) [inline src/sections/Header.test.js:20]: Keep the Header component in one line. Please check the same thing all over the file.

File: src/pages/Decision/Decision.test.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { Store } from '../../redux/store';
+import Decision from './index';
+
+test("Component should be rendered with out crash", () => {
+    render(<Provider store={Store}>
+        <Decision />
```
Comment (NaveenReliable) [inline src/pages/Decision/Decision.test.jsx:8]: Follow proper indentation here.

File: src/pages/Decision/Decision.test.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    </Provider>)
+})
+
+test("Check text of Heading should be rendered", () => {
+    render(<Provider store={Store}>
+        <Decision />
```
Comment (NaveenReliable) [inline src/pages/Decision/Decision.test.jsx:14]: Follow proper indentation here.

File: src/pages/Decision/Decision.test.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    </Provider>);
+    const heading = screen.getByText(/Today\'s Top 5 Decisions/i);
+    expect(heading).toBeInTheDocument()
+})
+
+test("Check text if No data Available ", () => {
+    render(<Provider store={Store}>
```
Comment (NaveenReliable) [inline src/pages/Decision/Decision.test.jsx:21]: Follow proper indentation here.

File: src/sections/Header.test.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
+import { render, screen } from '@testing-library/react';
+import Header from './Header';
+
+describe("Test the Header Component", () => {
+    test("render the component without Crash", () => {
+        render(
+            <Header />
```
[the remark this answers is not in the document]
Comment (AbhijitP98) [reply src/sections/Header.test.js]: changed to one line

File: src/sections/Header.test.js (hunk 1 above)
Comment (AbhijitP98) [reply src/sections/Header.test.js]: changed to one line for all

File: src/pages/Decision/Decision.test.jsx (hunk 2 above)
Comment (AbhijitP98) [reply src/pages/Decision/Decision.test.jsx]: added correct order

File: src/pages/Decision/Decision.test.jsx (hunk 3 above)
Comment (AbhijitP98) [reply src/pages/Decision/Decision.test.jsx]: added correct order

File: src/pages/Decision/Decision.test.jsx (hunk 4 above)
Comment (AbhijitP98) [reply src/pages/Decision/Decision.test.jsx]: added correct order

**Pull request merged**